content/api/api-events.md

```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
 alwaysopen: false
 pre: ""
 ---
-The events API allows you to consume events generated by CloudWisdom. You can either grab a list of events generated by CloudWisdom or grab an individual event by its ID. To explore CloudWisdom’s REST API and test some requests, go here.
-
-**Request Header**
-
-| Header Name | Header Value |
-|--------------|------------------|
-| Content-Type | application/json |
-
-## POST
-### POST to /ingest/events/{apiId}
-Creates an external event within CloudWisdom.
-
-```
-[
-  {
-    "source": "My Custom Datasource",
-    "title": "Server message 2",
-    "tags": [
-      {
-        "name": "Event Tag",
-        "value": "External"
-      }
-    ],
-    "type": "INFO",
-    "data": {
-      "elementId": "us-east-1:i-123a4567",
-      "level": "WARNING",
-      "message": "WARNING: Event is triggering!"
+
+## About the Events API
+
+
+## GET from /events
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/events/getEventsUsingGET_1" theme="info" >}} GET {{< /button >}} Use this endpoint to get details on events and their associated policies/elements.
+
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|-------------|----------------|---------------|---------------------------------------------------------------|
+| duration | query | string | Follows ISO8601 duration format. Example options include: PT1h, PT1m, PT10s, P3D |
+| startTime | query | date-time | Start time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| endTime | query | date-time | End time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| category | query | Array[string] | Defines event category; valid inputs are INFO, WARNING, or CRITICAL. |
+| elementId | query | Array[string] | Element Ids can be found in by selecting an element in CloudWisdom and viewing the URL (/#/inventory/6bdf4fd1-1111-1111-1c11-17eb5f628e46) |
+| elementName | query | Array[string] | Name of element. |
+| elementType | query | Array[string] | Type of element; examples include EC2, ALB, SERVER. |
+| elementTags | query | string | Tags associated to an element. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
+| eventTags | query | string | You can tag events to enable filtering in CloudWisdom by passing in keys and values. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
+| policy | query | Array[string] | Policy Id. |
+| isExternal | query | boolean |  |
```

Comment:
isExternal description is missing. It's whether or not the event is an external event.